### PR TITLE
Add What's New section for 3.8 in docs

### LINKED ISSUE #3242: Update what's new for next release

### PR DESCRIPTION
**Pull request by jessica-mitchell**
Content added for what's new section for 3.8 
NOTE: that the link for the performance benchmarks mentioned here is only available once PR #3244 is merged! 

Fixes #3242

### CODE REVIEW

File: doc/htmldoc/whats_new/v3.8/index.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+:ref:`list of updates for previous releases in the 3.x series <whats_new>`.
+
+
+NMDA dynamics after Brunel and Wang (2001)
+------------------------------------------
+
+A new integrate-and-fire model allows for the use
+simplified NMDA dynamics.
+It contains exponential conductance-based AMPA and GABA synapses along with NMDA synapses
+weighted such that it approximated the original linear dynamics.
```
Comment (diesmann) [inline doc/htmldoc/whats_new/v3.8/index.rst:23]: approximates

File: doc/htmldoc/whats_new/v3.8/index.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+The main page of the docs now contains a diagram highlighting the
+fundamental components of creating network simulations in NEST.
+
+* :doc:`/index`
+
+
+NEST performance benchmarks
+---------------------------
+
+You can now view the results from experiments in
```
Comment (diesmann) [inline doc/htmldoc/whats_new/v3.8/index.rst:43]: You can now view results of this release in benchmarks of

File: doc/htmldoc/whats_new/v3.8/index.rst (hunk 1 above)
Comment (jessica-mitchell) [inline doc/htmldoc/whats_new/v3.8/index.rst:23]:
```suggestion
weighted such that it approximates the original linear dynamics.
```

File: doc/htmldoc/whats_new/v3.8/index.rst (hunk 2 above)
Comment (jessica-mitchell) [inline doc/htmldoc/whats_new/v3.8/index.rst:43]:
```suggestion
You can now view results of this release in benchmarks of
```

**Comment by jessica-mitchell** [comment]
Merging with approval in meeting

**Pull request merged**